SConstruct

```diff
@@ -196,7 +196,7 @@ elif env_base["p"] != "":
     selected_platform = env_base["p"]
 else:
     # Missing `platform` argument, try to detect platform automatically
-    if sys.platform.startswith("linux"):
+    if sys.platform.startswith("linux") or sys.platform.startswith("openbsd"):
```

Comment:
Could we also detect other BSDs such as FreeBSD while we're at it?

Reply:
Sure :)

I checked on my server and python recognizes freebsd 1.3 as "freebsd13".  I've added also NetBSD and DragonFlyBSD while at it but I don't have access to those system.